devtools-har-extension/contentscript.js

```diff
@@ -0,0 +1,31 @@
+/* global chrome */
+(function (window, chrome) {
+  window.__sendPageTimings = function () {
+    var page = {
+      'pageTimings': {
+        'onContentLoad': window.performance.timing.domContentLoadedEventEnd - window.performance.timing.fetchStart,
+        'onLoad': window.performance.timing.loadEventEnd - window.performance.timing.fetchStart
+      },
+      'startedDateTime': new Date(window.performance.timing.fetchStart).toISOString(),
+      'title': window.location.href
+    }
```

Comment:
I can change the implementation to send the entire performance object, but the numbers will be discarded at the devtools-har-extension level, since it will only get the necessary values defined by the HAR specification to save in the har object.

I don't have information on how to consume the performance values. I did reverse engineering to find out what values I should use to have the same values that the network API return on the default getHAR method.